packages/ui/src/app/pages/Forum/ForumWatchlist.tsx

```diff
@@ -1,38 +1,45 @@
-import React, { useState } from 'react'
+import React from 'react'
+import styled from 'styled-components'
 
 import { PageLayout } from '@/app/components/PageLayout'
-import { ForumThreadOrderByInput } from '@/common/api/queries'
+import { HorizontalScroller } from '@/common/components/HorizontalScroller/HorizontalScroller'
+import { RowGapBlock } from '@/common/components/page/PageContent'
 import { PageTitle } from '@/common/components/page/PageTitle'
 import { useRefetchQueries } from '@/common/hooks/useRefetchQueries'
-import { useSort } from '@/common/hooks/useSort'
 import { MILLISECONDS_PER_BLOCK } from '@/common/model/formatters'
 import { ForumPageHeader } from '@/forum/components/ForumPageHeader'
-import { ThreadList } from '@/forum/components/threads/ThreadList'
+import { ThreadCard } from '@/forum/components/ThreadCard/ThreadCard'
+import { ThreadCardSkeleton } from '@/forum/components/ThreadCard/ThreadCardSkeleton'
 import { useWatchlistedThreads } from '@/forum/hooks/useWatchlistedThreads'
 
 import { ForumTabs } from './components/ForumTabs'
 
 export const ForumWatchlist = () => {
-  const { getSortProps } = useSort<ForumThreadOrderByInput>('updatedAt')
-  const [currentPage, setCurrentPage] = useState(1)
-  const { threads, pageCount, isLoading } = useWatchlistedThreads({ page: currentPage, threadsPerPage: 2 })
+  const { threads, isLoading } = useWatchlistedThreads({ threadsPerPage: -1 })
   const isRefetched = useRefetchQueries({
     interval: MILLISECONDS_PER_BLOCK,
     include: ['GetForumThreads', 'GetForumThreadsCount'],
   })
 
+  const loading = isLoading && !isRefetched
+
   const displayThreads = () => {
     return (
-      <ThreadList
-        threads={threads}
-        isLoading={isLoading && !isRefetched}
-        getSortProps={getSortProps}
-        type="card"
-        pageCount={pageCount}
-        setPage={setCurrentPage}
-        page={currentPage}
-        watchlistButton
-      />
+      <RowGapBlock gap={20}>
+        {!loading ? (
+          <HorizontalScroller
+            items={
+              threads.length ? (
+                threads.map((thread) => <StyledThreadCard thread={thread} watchlistButton />)
+              ) : (
+                <div>No threads found</div>
+              )
+            }
+          />
+        ) : (
+          <HorizontalScroller items={<ThreadCardSkeleton count={10} />} />
+        )}
+      </RowGapBlock>
```

Comment:
The code looks good but for me (both on Firefox and Chrome) it shows like this when there are less than 6 thread in my watchlist:
![image](https://user-images.githubusercontent.com/6571453/218739759-40be3b7a-4176-4c1d-994f-fd7bac81c333.png)

But I guess this can be handled in another issue...